### DISCUSSION TITLE
Multi Domain Next Js Prisma Next-Auth

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#10342] [Q&A]
Hello, 

Techno stack : next js, prisma, postgresql, next-auth

My project uses one single database containing all info ( user, session, project specific data).
Ideally I would like to have the following website structure : 

- admin.mysite.com : admin portal , only admin can loggin ,  admin specific information should be fetched from db
- user.mysite.com : admin portal , only users can loggin ,  user specific information should be fetched from db
- api.mysite.com : public information (project specific data) should be accssible publicly

With next JS, we can not work with different subdomains so each site must be separated( please let me know if I am wrong)
So I am struggling about how to organize my site to avoid prisma code redundancy and ensure security. 

I am thinking to put all the prisma logic and set up all needed api's in apy.mysite.com , but is there a way to ensure security easily between two sub domains (e.g. can I check at api.mysite.com that an admin requesting his profile picture is indeed logged as admin) ? 

Thank you so much in advance for your help 🤝

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
